### DISCUSSION TITLE
Würstchen support

### DISCUSSION THREAD

**Discussion by yesbroc** [lllyasviel/Fooocus#649] [Ideas]
faster inference times apparently, https://huggingface.co/blog/wuerstchen

**Comment by AkoZ** [comment]
there : https://github.com/dome272/wuerstchen/ 
paper: https://arxiv.org/abs/2306.00637 and https://huggingface.co/warp-ai/wuerstchen-prior-model-interpolated/tree/main for files
Official doc: https://huggingface.co/docs/diffusers/main/en/api/pipelines/wuerstchen
